Binary files orig/AneuFinder/src/.Makevars.swp and patched/AneuFinder/src/.Makevars.swp differ
diff -Nru orig/AneuFinder/src/Makevars patched/AneuFinder/src/Makevars
--- orig/AneuFinder/src/Makevars	1970-01-01 01:00:00.000000000 +0100
+++ patched/AneuFinder/src/Makevars	2024-03-16 11:14:41.281205100 +0100
@@ -0,0 +1 @@
+PKG_CPPFLAGS = -DR_NO_REMAP
